.gitignore

```diff
@@ -13,6 +13,10 @@
 /Cargo.lock
 /.vscode
 /_*
+/zola/public
```

Comment:
Add readme with **minimal** instruction and explaining of files tree structure
use command `tree` to get tree of dirs

Reply:
updated in last commit